### DISCUSSION TITLE
Ventura - long delays (30-60s) saving mail attachments when MountainDuck is running

### DISCUSSION THREAD

**Discussion by pjd-nu** [iterate-ch/cyberduck#14785] [Q&A]
A week or so ago I started having trouble with horribly long delays (with spinning beachball of death) when saving mail attachments from either Apple Mail or Outlook. 

I've just verified that if I quit MountainDuck the delays go away, and they reappear when I start it up again.

At some point I'd be willing to look into this some to give more debug info, but I'm swamped now, so I'm just turning MountainDuck off and getting on with my work.

**Comment by TedSlaterDev** [comment]
I've been having similar problems with MountainDuck, and am looking forward to a bug fix.

**Comment by pjd-nu** [comment]
Turns out it only happens if you don’t have finder extensions installed. I think.

Sent from my iPhone

On Jun 14, 2023, at 1:47 PM, Ted Slater ***@***.***> wrote:

﻿

I've been having similar problems with MountainDuck, and am looking forward to a bug fix.

**Reply by TedSlaterDev** [reply]
I appreciate the quick reply! I've confirmed that the Mountain Duck finder extension is enabled.   :-/

https://docs.mountainduck.io/mountainduck/installation/
